staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -419,6 +421,11 @@ func (f *StorageFactoryRestOptionsFactory) GetRESTOptions(resource schema.GroupR
 		} else {
 			klog.V(3).InfoS("Using watch cache", "resource", resource)
 			ret.Decorator = genericregistry.StorageWithCacher()
+
+			if utilfeature.DefaultFeatureGate.Enabled(features.WatchFromStorageWithoutPrevKV) {
```

Comment:
This is not needed, apiserver logs command line flags including feature flags that were set.

Reply:
Right, so it will generate a lot of logs that all will say the same value. Whether `WatchFromStorageWithoutPrevKV` really works is based on fact whether watch cache is enabled and feature flag is set.